app/assets/javascripts/discourse/app/components/user-nav/dropdown-list.hbs

```diff
@@ -0,0 +1,18 @@
+<li class={{concat @class " user-nav-dropdown-list-item"}}>
+  <button type="button" class={{this.buttonClass}} {{on "click" this.toggleList}}>
+    {{d-icon @icon}}
```

Comment:
How come we don't use the form `DIcon` for this like we do with `DNavigationItem`?

Reply:
Is it because it's a helper, not a component?

Reply:
Yup it is a helper although I wonder why it isn't a component.